styles/labs.css

```diff
@@ -274,6 +274,26 @@ button.success {
     box-shadow: 0 4px 12px rgba(0, 0, 0, 0.15);
 }
 
+.feature-card-disabled .feature-image {
```

Comment:
I am changing opacity of items one-by-one, because feedback button needs to keep the original opacity. If I just change the opacity of the parent element, it is impossible to overwrite it just for the feedback button later.